src/Features/Lsif/Generator/Graph/Capabilities.cs

```diff
@@ -44,7 +48,9 @@ internal sealed class Capabilities : Vertex
             bool typeDefinitionProvider,
             bool documentSymbolProvider,
             bool foldingRangeProvider,
-            bool diagnosticProvider)
+            bool diagnosticProvider,
+            IReadOnlyList<string> semanticTokenTypes,
+            IReadOnlyList<string> semanticTokenModifiers)
```

Comment:
Any reason this doesn't just take a SemanticTokensCapabilities so that way any additional members don't have to be threaded through?

Reply:
The thought process was that I don't expect Roslyn to ever use any other optional semantic tokens capabilities. In LSP, for example, or at least not in a way that needs passed in. Most of those additional capabilities are for things like alternate token encodings, which are theoretically possible but not used by any editor or LSP (and now LSIF) generator.

Reply:
Updated, as you suggested.